uservice_ccutter/plugins/substitute.py

```diff
@@ -16,6 +16,11 @@ def substitute(templatetype, auth, inputdict):
     one field that's guaranteed to be present so that we can use its hook to
     drive creation of the github_* fields.
     """
+    log = None
+    if "_logger_" in inputdict:
+        log = inputdict["_logger_"]
```

Comment:
Since it seems like this is reuseable, just collapse it into a single method and call that.